src/main/java/com/echobox/api/linkedin/client/DefaultWebRequestor.java

```diff
@@ -228,20 +227,13 @@ public Response executePost(String url, String parameters, String jsonBody) thro
     return executePost(url, parameters, jsonBody, null, new BinaryAttachment[0]);
   }
 
+  // CPD-OFF
   @Override
   public Response executePost(String url, String parameters, String jsonBody,
       Map<String, String> headers,
       BinaryAttachment... binaryAttachments)
       throws IOException {
     
-    if (LOGGER.isDebugEnabled()) {
-      LOGGER.debug("Executing a POST to " + url + " with parameters "
```

Comment:
just thinking, is it still worth logging at the start of the method. Just in case the rest of the method fails for some reason?

I don't think this is necessary, as long as `hadnleException(ex);` in the catch block logs something.

Reply:
The log has been moved just a bit further down, once the request has been setup and just before the API call has been made. If something fails before the log, it would be around setting up the request, which the exception handling should catch and display. Every API call will be logged whether it fails or succeeds.